### ISSUE TITLE
Strange XMLHttpRequest error

### ISSUE THREAD

**Issue by evanjmg** [angular/zone.js#787]
with the latest zone.js-0.8.11 - I'm getting with AOT  - Angular 4.0.2 - running ngc -p && then webpack in production, I can reproduce this locally and on heroku 
```
 Uncaught Error: eventTask XMLHttpRequest.addEventListener:readystatechange': can not transition to 'running', expecting state 'scheduled', was 'notScheduled'.
    at t._transitionTo (polyfills.6d82844656613bc57a6b.bundle.js:1)
    at r.runTask (polyfills.6d82844656613bc57a6b.bundle.js:1)
    at XMLHttpRequest.invoke (polyfills.6d82844656613bc57a6b.bundle.js:1)

at this line in zonejs ->                         t.prototype._transitionTo = function(t, n, e) {
                            if (this._state !== n && this._state !== e)
                                throw new Error(this.type + " '" + this.source + "': can not transition to '" + t + "', expecting state '" + n + "'" + (e ? " or '" + e + "'" : "") + ", was '" + this._state + "'.");
                            this._state = t,
                            t == _ && (this._zoneDelegates = null)
                        }
```
I'm assuming this is related to https://github.com/angular/zone.js/issues/530 - but shouldn't it be open? It's pretty easy to reproduce

**Comment by JiaLiPassion** [comment]
@evanjmg, thanks for posting the issue, can you provide a reproduce repo? and can you try it without AOT?

**Comment by evanjmg** [comment]
seems upgrading Angular to the latest fixes it ->  Angular 4.1.3, maybe it's compatibility issues? I'll close this issue for now as it shouldn't a problem if you have the latest.
